### PR TITLE
chore: [CCM-8061]: Microfrontend types version update

### PR DESCRIPTION
**Pull request by jenil-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by jenil-harness** [comment]
trigger cypress

File: src/microfrontends/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.0.2",
+  "version": "1.0.4",
```
Comment (knagurski) [inline src/microfrontends/package.json:3]: Since this isn't a bug fix, it should be a minor, not a patch.
```suggestion
  "version": "1.1.0",
```

Comment (jenil-harness) [reply src/microfrontends/package.json:3]: This is an update.
We have published packages till 1.0.4 and it has not got updated here.
I am just updating the version

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
👍🏻

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**